microsite/docusaurus.config.ts

```diff
@@ -266,7 +276,27 @@ const config: Config = {
         ratingMode: 'stars',
       },
     ],
+    [
+      'docusaurus-plugin-openapi-docs',
```

Comment:
Ran into a minor cosmetic issue:

![image](https://github.com/user-attachments/assets/e41c27fa-d531-4f52-afa4-cec0d838c6fd)

The double `//` goes away if I remove the leading `/` from the generated `.mdx` files, but perhaps there's some config somewhere to avoid the other `/`. Either way not a blocking issue, fine to merge without a fix for this.